pom.xml

```diff
@@ -1,25 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ JBoss, Home of Professional Open Source.
```

Comment:
Do you want all the whitespace change here?

I guess it's ok but maybe in a separate commit?

Reply:
no i'll remove those